doc/source/whatsnew/v0.23.3.txt

```diff
@@ -1,7 +0,0 @@
-.. _whatsnew_0233:
-
-v0.23.3 (July 7, 2018)
-----------------------
```

Comment:
Not sure v.0.23.3 should be removed completely? Leaves a puzzling gap between v.0.23.2 and v.0.23.4 otherwise